wordpress-mobile/WordPress-Android
# Wire "Remind Me" button to open site picker or reminders flow

Fixes #16234

To test:
The action that triggers this dialog is not implemented yet, so in order to test it you just have to show it anywhere in the app. Nothing has to be injected to the dialog class.
e.g. in `WPMainActivity`: ```BloggingPromptsOnboardingDialogFragment.newInstance().show(getSupportFragmentManager(), BloggingPromptsOnboardingDialogFragment.TAG);```

After showing the dialog, click in the "Remind Me" button: if there is more than 1 site, it should open the site picker. After selecting the site, it goes back to the dialog and does nothing because the flow that should de opened is not implemented yet. If there's only 1 site, it does nothing for the same reason.

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
--

3. What automated tests I added (or what prevented me from doing so)
Updated Blogging Prompts introduction dialog unit tests

PR submission checklist:
